### PR TITLE
Update bib.md

### PR DESCRIPTION
**Pull request by PRUBHTEJ**
Correcting a small grammatical error & a typo.

### CODE REVIEW

**Comment by PRUBHTEJ** [comment]
@googlebot I signed it!

**Comment by PRUBHTEJ** [comment]
Requesting @lamberta or @MarkDaoust or @yashk2810 to approve.

File: site/en/about/bib.md [hunk 1]
```diff
@@ -107,13 +107,13 @@ TensorFlow: Large-scale machine learning on heterogeneous systems,
 
 [Access this white paper.](https://www.usenix.org/system/files/conference/osdi16/osdi16-abadi.pdf)
 
-**Abstract:** TensorFlow is a machine learning system that operates at
+**Abstract:** TensorFlow is a machine learning system that operates at a
```
Comment (lamberta) [inline site/en/about/bib.md:110]: Can leave this as-is. It's idiomatic and would rather it match the published abstract: https://www.usenix.org/system/files/conference/osdi16/osdi16-abadi.pdf

**Comment by PRUBHTEJ** [comment]
Should I close the PR then?

**Comment by lamberta** [comment]
Can you update your commit to remove the first change and keep the second change? Then we can merge. Thanks

**Comment by PRUBHTEJ** [comment]
Done that, @lamberta

**Comment by PRUBHTEJ** [comment]
> Thanks for the fix!
My pleasure!!

**Comment by PRUBHTEJ** [comment]
@lamberta , The "import/copybara" check is still pending for almost 12 hours. I couldn't understand the logic behind it though.

**Comment by lamberta** [comment]
Yep, PRs need to be approved internally as well. And we tend to do that in batches, so can take a little time to merge.

**Pull request merged**